src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -106,9 +106,20 @@ public static bool IsPrivilegedProcess
         public static bool IsReleaseLibrary(Assembly assembly) => !IsDebuggable(assembly);
         public static bool IsDebugLibrary(Assembly assembly) => IsDebuggable(assembly);
 
-        // For use as needed on tests that time out when run on a Debug runtime.
+        // For use as needed on tests that time out when run on a Debug or Checked runtime.
         // Not relevant for timeouts on external activities, such as network timeouts.
-        public static int SlowRuntimeTimeoutModifier = (PlatformDetection.IsDebugRuntime ? 5 : 1);
+        public static int SlowRuntimeTimeoutModifier
+        {
+            get
+            {
+                if (IsReleaseRuntime)
+                    return 1;
+                if (IsRiscV64Process)
+                    return IsDebugRuntime? 10 : 2;
+                else
+                    return IsDebugRuntime? 5 : 1;
+            }
+        }
```

Comment:
Mismatch between comment and condition? IMO IsDebugRuntime means just Debug because there's a separate IsCheckedRuntime.

Reply:
>IMO IsDebugRuntime means just Debug because there's a separate IsCheckedRuntime.

That's correct.

> Mismatch between comment and condition?

Before patch _SlowRuntimeTimeoutModifier_ was adjusted only for Debug build. Now _SlowRuntimeTimeoutModifier_ is adjusted for Debug and Checked builds for RISC-V. I don't see how it's mismatched. Or I don't follow.

Reply:
What I mean is now SlowRuntimeTimeoutModifier for RV is:
```c#
if (IsRiscV64Process)
                     return IsDebugRuntime? 10 : 2;
```
No IsCheckedRuntime in code, but comment says "or Checked".

Reply:
We bail out in previous condition for release build, so at this point when IsDebugRuntime is false, IsReleaseRuntime is false too, which means IsCheckedRuntime must be true returning 2, no?